sentences/hr/_common.yaml

```diff
@@ -1,12 +1,62 @@
 language: hr
 responses:
   errors:
-    no_intent: TODO Sorry, I couldn't understand that
-    no_area: TODO No area named {{ area }}
-    no_domain: TODO {{ area }} does not contain a {{ domain }}
-    no_device_class: TODO {{ area }} does not contain a {{ device_class }}
-    no_entity: TODO No device or entity named {{ entity }}
-    handle_error: TODO An unexpected error occurred while handling the intent
-lists: {}
-expansion_rules: {}
-skip_words: []
+    no_intent: "Nažalost, nisam vas razumjeo"
+    no_area: "Prostorija sa imenom {{ area }} ne postoji"
+    no_domain: "Prostorija {{ area }} ne postoji u sklopu {{ domain }}"
```

Comment:
> It should be translated as "Prostorija {{ area }} ne sadrži {{ domain }}" to match original. The same order is used in the next example.

You can't always follow original, languages doesn't work that way. "Prostorija ne sadrži" just sounds strange, have you ever heard "Sadrži li ova prostorija koga?"

Reply:
Yeah sure, I get that languages aren't 1:1, but I think we are not aligned on the intent of the sentence.

What is an example value of `{{ domain }}`? Perhaps I misunderstood something here, isn't it supposed to be like climate, cover, light, etc. See here: https://github.com/home-assistant/intents/blob/8e2a3ee74fbb41a173b83b6bfdf5b31d3d0fbba4/intents.yaml#L16
This would make the sentence sound like e.g. "Kitchen does not contain a light". Translated "Kuhinja ne sadrži svjetlo." and the current translation would be "Prstorija kuhinja ne postoji u sklopu svjetlo." which makes no sense.
This also makes me think the word 'Prostorija' (Room) is superfluous as it also does not appear in the original and the sentence "Prostorija kuhinja ne sadrži svjetlo." Sounds a bit unnatural.

Should we go with `{{ area }} ne sadrži {{ domain }}` and `{{ area }} ne sadrži {{ device_class }}`